src/streamlit_app.py

```diff
@@ -447,19 +582,24 @@ def process_response(response):
 
 # @st.fragment()
 def show_query_result(parent, df, resp_val, sql):
-    data_tab, explain_tab, hd_sql_tab = parent.tabs(["Data", "Explain", "SQL"])
+    data_tab, chart_tab, explain_tab, hd_sql_tab = parent.tabs(
+        ["Data", "Chart", "Explain", "SQL"]
```

Comment:
i think that the chart and data are better together - or at least, the chart to come first imho

Reply:
Fixed